composer.json

```diff
@@ -12,9 +12,11 @@
 	"prefer-stable": true,
 	"minimum-stability": "dev",
 	"require": {
+		"ext-hash": "*",
+		"ext-json": "*",
```

Comment:
`ext-json` was already needed as it is a [WordPress requirement](https://make.wordpress.org/hosting/handbook/server-environment/#php-extensions) before. Since I added more json_*() function usage I also updated composer.json.

`ext-hash` was an optional requirement (listed on the same page) and we are using `hash_hmac()` to generate the `sha256` signature required for an HS256 JWT